### ISSUE TITLE
Error while training the model

### ISSUE THREAD

**Issue by aberrya** [aiformankind/wildfire-smoke-detection-camera#1]
**Issue**
Training get  **Killed** with error.

**Steps:**
1) Build Docker as specified in steps in README.md
2) Start training with command
`python models/research/object_detection/legacy/train.py --logtostderr --train_dir=/tensorflow/train/ --pipeline_config_path=/tensorflow/train/faster_rcnn_resnet101.config`

**Results:**
It get **Killed** after following error.

`INFO:tensorflow:Saving checkpoint to path /tensorflow/train/model.ckpt
INFO:tensorflow:Starting Queues.
2020-06-29 21:41:19.528436: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node Preprocessor/ResizeToRange/strided_slice_3. Error: Pack node (Preprocessor/ResizeToRange/stack_2) axis attribute is out of bounds: 0
2020-06-29 21:41:20.569653: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node Preprocessor/ResizeToRange/strided_slice_3. Error: Pack node (Preprocessor/ResizeToRange/stack_2) axis attribute is out of bounds: 0
INFO:tensorflow:global_step/sec: 0
INFO:tensorflow:Recording summary at step 0.
2020-06-29 21:41:33.217339: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node Preprocessor/ResizeToRange/strided_slice_3. Error: Pack node (Preprocessor/ResizeToRange/stack_2) axis attribute is out of bounds: 0
2020-06-29 21:41:37.386351: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node Preprocessor/ResizeToRange/strided_slice_3. Error: Pack node (Preprocessor/ResizeToRange/stack_2) axis attribute is out of bounds: 0
Killed`

NOTE: I checked on net for above error and found few folks were able to proceed even with these error also. Some says these are just warning not errors.
But in this case, training get killed immediately after these errors.

OS - MacBook Pro with 16GB RAM

**Comment by aberrya** [comment]
Issue **resolved** by increasing docker  memory to 8GB.
